src/js/button.js

```diff
@@ -26,24 +26,14 @@ class Button extends ClickableComponent {
    * @method createEl
    */
   createEl(tag = 'button', props = {}, attributes = {}) {
+    if (tag !== 'button') {
+      throw new Error('Buttons must be created with a "button" tag.');
```

Comment:
making it would be good to not throw? Doesn't seem like a fatal error to me, but I could be persuaded either way.